File: 2018/11/faktor/faktor.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let A, I;

rl.on("line", line => {
  [A, I] = line.split(" ").map(c => parseInt(c));
});

rl.on("close", () => {
  const discount = 0.99;
  const result = Math.ceil(A * (I - discount));
  console.log(result);
});

File: 2018/11/faktor/faktor.cc
// clear; make clean; make faktor; cat samples/faktor.dummy.1.in | ./faktor

#include <iostream>
#include <math.h>

int main()
{
    int A, I;
    float discount = 0.99;
    std::cin >> A >> I;
    int result = ceil(A * (I - discount));
    std::cout << result << std::endl;
}
